functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/DebugMode325MavenSmartTestingFunctionalTest.java

```diff
@@ -13,12 +16,14 @@
 import static org.arquillian.smart.testing.ftest.configuration.CustomAssertions.assertThatAllBuiltSubmodulesContainBuildArtifact;
 import static org.arquillian.smart.testing.ftest.testbed.TestRepository.testRepository;
 import static org.arquillian.smart.testing.ftest.testbed.configuration.Mode.ORDERING;
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Mode.SELECTING;
 import static org.arquillian.smart.testing.ftest.testbed.configuration.Strategy.AFFECTED;
+import static org.arquillian.smart.testing.scm.ScmRunnerProperties.SCM_LAST_CHANGES;
 import static org.arquillian.smart.testing.scm.ScmRunnerProperties.SCM_RANGE_HEAD;
 import static org.arquillian.smart.testing.scm.ScmRunnerProperties.SCM_RANGE_TAIL;
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class DebugModeSmartTestingFunctionalTest {
+public class DebugMode325MavenSmartTestingFunctionalTest {
```

Comment:
I can see class name changed from `DebugModeSmartTestingFunctionalTest` to  `DebugMode325MavenSmartTestingFunctionalTest`. But looks like this class also contains another test with name `should_show_debug_logs_when_maven_is_executed_in_debug_mode` where we are not using maven version `3.2.5`. We should not change this class name as I believe we are already defining maven version in test method name

Reply:
The rename was meant to point out that this class is testing also different Maven version - easy to find.
The reason why I added it into existing test is that I didn't want to add another functional test that would make the test execution longer. 
As for the name - well, it is quite hard to describe all test method variants in the class name.

Reply:
But maybe we can create separate class with name `DebugModeMaven325SmartTestingFunctionalTest` & move it there this existing tests

Reply:
I was thinking about it as well, but I'd like to keep all debug related tests inside of one test class. How about rename it to `DebugModeAnd325MavenFunctionalTest`?

Reply:
Can we make it `DebugModeAndMaven325FunctionalTest` ?